src/plugins/intel_cpu/src/nodes/executors/subgraph.hpp

```diff
@@ -98,10 +99,42 @@ class SubgraphBaseExecutor {
     std::vector<ptrdiff_t> m_start_offset_out = {};
 };
 
+class SubgraphSpecializedBaseExecutor {
+public:
+    SubgraphSpecializedBaseExecutor(const std::set<size_t>& external_ptrs_idces, size_t in_num) {
+        precompute_ptr_mappings(external_ptrs_idces, in_num, m_src_ptr_mappings, m_external_ptr_mappings);
+    };
+
```

Comment:
Could you please add virtual default destructor? Then we can remove default virtual destructors from static and dynamic children-classes